internal/provider/resource_gitlab_project_runner.go

```diff
@@ -0,0 +1,120 @@
+package provider
+
+import (
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+var _ = registerResource("gitlab_project_runner", func() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabProjectRunnerCreate,
```

Comment:
You'll want to add a description here (as a "Description" field in the schema.Resource struct) detailing what your resource does. Something like "the gitlab_project_runner resource allows users to manage the linking between a project as a runner instance". Our linter will require you to add an upstream API link, similar to this: https://github.com/gitlabhq/terraform-provider-gitlab/blob/main/internal/provider/resource_gitlab_group.go#L24

Reply:
I've addressed this in the latest version